Code/EnginePlugins/JoltPlugin/System/JoltWorldModule.cpp

```diff
@@ -762,5 +772,292 @@ void ezJoltWorldModule::UpdateConstraints()
   m_RequireUpdate.Clear();
 }
 
+ezAtomicInteger32 s_iColMeshVisGeoCounter;
+
+ezView* ezJoltWorldModule::GetMainView() const
+{
+  for (ezViewHandle hView : ezRenderWorld::GetMainViews())
+  {
+    ezView* pView;
+    if (!ezRenderWorld::TryGetView(hView, pView))
+      continue;
+
+    if (pView->GetWorld() != GetWorld())
+      continue;
+
+    if (pView->GetCameraUsageHint() == ezCameraUsageHint::EditorView ||
+        pView->GetCameraUsageHint() == ezCameraUsageHint::MainView)
+    {
+      return pView;
+    }
+  }
+
+  return nullptr;
+}
+
+struct DebugVis
+{
+  const char* m_szMaterial = nullptr;
+  ezColor m_Color;
+};
+
+// "{ e6367876-ddb5-4149-ba80-180af553d463 }" = Data/Base/Materials/Common/PhysicsColliders.ezMaterialAsset
+
+static const DebugVis s_Vis[ezPhysicsShapeType::Count][2] =
+  {
+    // Static
+    {
+      {"{ e6367876-ddb5-4149-ba80-180af553d463 }", ezColor::LightSkyBlue}, // non-kinematic
+      {"{ e6367876-ddb5-4149-ba80-180af553d463 }", ezColor::Red}           // kinematic
+    },
+
+    // Dynamic
+    {
+      {"{ e6367876-ddb5-4149-ba80-180af553d463 }", ezColor::Gold},      // non-kinematic
+      {"{ e6367876-ddb5-4149-ba80-180af553d463 }", ezColor::DodgerBlue} // kinematic
+    },
+
+    // Query
+    {
+      {"{ ca43dda3-a28c-41fe-ae20-419182e56f87 }", ezColor::GreenYellow.WithAlpha(0.5f)}, // non-kinematic
```

Comment:
Would be nice to know what this material is as you have documented it for the static/dynamic material already. Same for cloth below.